docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,119 @@
+# Release 407 (16 Feb 2023)
+
+## General
+
+* Add support for correlated queries involving a `VALUES` clause with a single
+  row. ({issue}`15989`)
+* Reduce memory usage for large schemas. This behavior can be enabled with the
+  `query.remote-task.enable-adaptive-request-size` configuration property and
+  configured with the `query.remote-task.max-request-size`,
+  `query.remote-task.request-size-headroom`, and
+  `query.remote-task.guaranteed-splits-per-task` configuration properties or
+  their respective session properties. ({issue}`15721`)
+* Improve concurrency when small, concurrent queries are run on a large cluster
+  by automatically determining how many nodes to use for distributed joins
+  and aggregations. This can be managed with the `query.max-hash-partition-count`
+  (renamed from `query.hash-partition-count`) and `query.min-hash-partition-count`
+  configuration properties. ({issue}`15489`)
+* Improve query memory tracking. ({issue}`15983`)
+* Improve memory usage accounting for queries with dynamic filters. ({issue}`16110`)
+* Improve query performance when a predicate evaluates to a null value. ({issue}`15744`)
+* Improve performance of queries with joins on the output of global
+  aggregations. ({issue}`15858`)
+* Improve performance of selective queries, queries that read a small number of
+  columns, and queries that process tables with large Parquet row groups or Orc
+  stripes. ({issue}`15579`)
+* Improve performance of queries with window functions. ({issue}`15994`)
+* Return an exit code of `100` when Trino crashes during startup. ({issue}`16113`)
+* Fix precision loss when converting `time` values with a precision higher than
+  three and `time with time zone` values with lower precision. ({issue}`15861`)
+* Fix potential incorrect results when a query fails but reports normal
```

Comment:
Why would a query fail and report normal completion? Or is the bug that it fails and reports normal completion?

If so, I'd rephrase as: "Fix potential incorrect results due to query reporting normal completion instead of failing"